<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="laJmeno">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="laPrijmeni">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="laTelefon">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionNovy"/>
    <addaction name="actionOtevrit"/>
    <addaction name="actionUlozit"/>
    <addaction name="separator"/>
    <addaction name="actionKonec"/>
   </widget>
   <widget class="QMenu" name="menuNastaven">
    <property name="title">
     <string>Nastavení</string>
    </property>
    <addaction name="actionNastaveni"/>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menuNastaven"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovy">
   <property name="text">
    <string>Nový</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOtevrit">
   <property name="text">
    <string>Otevřít</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUlozit">
   <property name="text">
    <string>Uložit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionKonec">
   <property name="text">
    <string>Konec</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNastaveni">
   <property name="text">
    <string>Nastavení</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionKonec</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
